altapay.php

```diff
@@ -771,6 +786,8 @@ private function processPaymentActions()
             exit();
         }
 
+        $reconciliation_identifier = getAltapayOrderReconciliationIdentifier($orderID);
```

Comment:
Although Shopify had a limitation about this, we should try, as much as we can, to have a different reconciliation identifier for each trasaction:
- capture reconciliationId1
- refund reconciliationId2
- refund reconciliationId3

This will ensure the merchant can reconcile individual transactions at ease.

Would this be possible?

Reply:
@emicha Yes. By default gateway automatically sets different reconciliation identifier for every transaction. This make it simple that we should not set it ourselves and rather show the different reconciliation identifiers from the gateway with their transaction types.

captured reconIdFromGateway1
refunded reconIdFromGateway2
refunded reconIdFromGateway3